Example/ReactiveDataDisplayManager/Table/Views/Cells/Generators/DiffableCellGenerator.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DiffableCellGenerator.swift
+//  ReactiveDataDisplayManagerExample
+//
+//  Created by Anton Eysner on 04.02.2021.
+//  Copyright © 2021 Alexander Kravchenkov. All rights reserved.
+//
+
+import ReactiveDataDisplayManager
+
+final class DiffableCellGenerator: BaseCellGenerator<TitleTableViewCell>, DiffableItemSource {
+
+    let uuid = UUID().uuidString
```

Comment:
Тут можно и private сделать, чтобы наружу id не торчал
```suggestion
    private let uuid = UUID().uuidString
```

Reply:
поправил